app/src/main/java/com/app/foodforyou/view/FoodListActivity.java

```diff
@@ -93,10 +93,10 @@ public void run() {
             }
         }).start();
 
-        adapter.setOnFoodItemClickListener(new RecommendDietFoodListAdapter.OnFoodItemClickListener() {
+        adapter.setOnFoodItemClickListener(new FoodListAdapter.OnFoodItemClickListener() {
             @Override
             public void onFoodItemClick(View view, int position) {
-                Intent intent = new Intent(getApplicationContext(), FoodRecipeDetails.class);
+                Intent intent = new Intent(getApplicationContext(), FoodRecipeDetailsActivity.class);
```

Comment:
activity please not getApplicationContext

Reply:
This is still using getApplicatiionContext, please change to activity context :)

Reply:
Thank you! Is this correct what you want?

` Intent intent = new Intent(getActivity(), FoodRecipeDetailsActivity.class);`

If it is correct, it seems impossible. there are only 3 methods.. could you let me know another way? :)
<img width="704" alt="image" src="https://user-images.githubusercontent.com/64777031/94443657-6395dd00-01e0-11eb-9ff0-57a296b80166.png">

Reply:
Since its activity then can just use 'this', getActivity will be used in fragment :)

Reply:
ahh got it. thank you! 
In this activity, it couldn't set "this" because intent is in the setOnFoodItemClickListener. 
so the intent variable should be declared global. but I've already set "mContext = this" so I used "mContext" instead of "this".
I fixed it. please look at it again. thanks a lot 👍

Reply:
Ahh in the case where context is inside a listener then use "activity.this", in this case FoodRecipeDetailsActivity.this :)